### ISSUE TITLE
OSError on import when Windows username contains non-Latin char

### ISSUE THREAD

**Issue by davemfish** [natcap/invest#1018]
We have a [report of this on the forum](https://community.naturalcapitalproject.org/t/requestion-for-obtaining-version-3-10-2/2876/7). It seems to happen on import of `scipy`, which happens pretty much right away for any `invest` cli call. So this effects Qt, Workbench, python.

It sounds like a scipy issue: https://github.com/scipy/scipy/issues/8850

Here's some logging from the Workbench: 

```
[15:17:56.760] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js] retry # 4
[15:17:56.836] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js] Traceback (most recent call last):
  File "natcap\invest\cli.py", line 18, in <module>

[15:17:56.837] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "natcap\invest\datastack.py", line 37, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "natcap\invest\utils.py", line 18, in <module>

[15:17:56.838] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pygeoprocessing\__init__.py", line 9, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load

[15:17:56.839] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "pygeoprocessing\geoprocessing.py", line 27, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked

[15:17:56.840] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "scipy\signal\__init__.py", line 331, in <module>

[15:17:56.840] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "scipy\signal\_peak_finding.py", line 8, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load

[15:17:56.841] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "scipy\stats\__init__.py", line 468, in <module>
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load

[15:17:56.842] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "PyInstaller\loader\pyimod03_importers.py", line 495, in exec_module
  File "scipy\stats\_rvs_sampling.py", line 3, in <module>
  File "unuran_wrapper.pyx", line 221, in init scipy.stats._unuran.unuran_wrapper

[15:17:56.842] [C:\Users\刀削面\AppData\Local\Programs\InVEST 3.11.0 Workbench\resources\app.asar\build\main\main.js]   File "unuran_wrapper.pyx", line 200, in scipy.stats._unuran.unuran_wrapper._setup_unuran
  File "messagestream.pyx", line 36, in scipy._lib.messagestream.MessageStream.__cinit__
OSError: Failed to open file b'C:\\Users\\\xe5\x88\x80\xe5\x89\x8a\xe9\x9d\xa2\\AppData\\Local\\Temp\\scipy-xyzs4454'
[1252] Failed to execute script 'cli' due to unhandled exception!
```

**Comment by dcdenu4** [comment]
Interesting that the issue you pointed to has been open since 2018. I'm surprised we wouldn't have seen this before...

**Comment by davemfish** [comment]
The scipy issue linked above suggests they fixed this for the `1.9.0` release.
